db/migrations/sdp-migrations/2024-11-01.1-disbursement-make-verification-field-nullable.sql

```diff
@@ -0,0 +1,7 @@
+-- This migration drops the verification_field NOT NULL constraint from the disbursements table.
+
+-- +migrate Up
+ALTER TABLE disbursements
+    ALTER COLUMN verification_field DROP NOT NULL;
+
+-- +migrate Down
```

Comment:
Just checking, but I assume this was done on purpose because we can't populate existing NULL values once we drop the constraint with a reasonable default ?

Reply:
Yes, that was my rationale as well.